justfile

```diff
@@ -11,13 +11,15 @@ alias qm := quick-modify
 # Installs the tools needed to develop
 install-tools:
 	cargo install cargo-binstall
+	cargo install tree-sitter-cli
```

Comment:
binstall not supported :( https://github.com/tree-sitter/tree-sitter/issues/4351